flink-runtime/pom.xml

```diff
@@ -117,6 +117,16 @@ under the License.
 		</dependency>
 
 		<dependency>
+			<groupId>org.slf4j</groupId>
+			<artifactId>slf4j-log4j12</artifactId>
```

Comment:
The problem I had was that I couldn't start the `TaskManagerRunner#main` from within Intellij, because the class `org.apache.log4j.Level` could not be found. If I'm not mistaken, then we filter out the log4j dependencies when building the dist packages.